wiki/Difficulties/id.md

```diff
@@ -0,0 +1,77 @@
+---
+tags:
+  - difficulty
+  - difficulty name
+  - SR
+  - star rating
+  - stars
+---
+
+# Kesulitan beatmap
+
+*Tidak sama dengan [Beatmaps](/wiki/Beatmaps).*
+
+**Kesulitan** beatmap menggambarkan tingkat keterampilan yang diperlukan untuk menyelesaikannya. Nama tiap tingkat dan *star rating* dari sebuah beatmap biasanya memberikan informasi yang cukup untuk mengetahui seberapa sulit sebuah beatmap.
+
+## Tingkat kesulitan
+
+*Lihat juga: [Penamaan Tingkat Kesulitan](/wiki/Ranking_Criteria/Difficulty_Naming)*
+
+Berikut adalah nama tingkat kesulitan yang biasa dimiliki dalam beatmap. Definisi dan persyaratan khusus dari masing - masing tingkat dapat ditemukan dalam [kriteria beatmap Ranked](/wiki/Ranking_Criteria) untuk masing - masing mode.
```

Comment:
just making sure is this "masing - masing" part correct? not sure what its function is in indonesian

Reply:
It is correctly placed, each mode means masing - masing mode in Indonesian

Reply:
each = masing masing